### DISCUSSION TITLE
Self hosted runner on macos can't find path or other env vars

### DISCUSSION THREAD

**Discussion by Gnts** [community/community#37888] [Actions]
Hi!

**Context:** I am attempting to setup a self hosted runner for github actions which I could use for building unity game builds using a macmini.
**Issue:** If I install the github runner as a [service](https://docs.github.com/en/actions/hosting-your-own-runners/configuring-the-self-hosted-runner-application-as-a-service?platform=mac), then my PATH and any other env variables are not available. For example git-lfs which I installed using brew.

**What I have tried:**
tried adding and logging variables from ~/.bash_profile, ~/.profile, ~/.zshrc, ~/.zprofile, /etc/profile, /etc/bashrc
restarting the whole machine
install and uninstall + start the service
add ~/.launchd.conf

Everything works just fine if I start the runner from terminal, but it all falls apart if I install it as a service.

**Comment by ossanna16** [comment]
Hi there @Gnts and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
